internal/providers/terraform/azure/util.go

```diff
@@ -114,3 +114,12 @@ func locationNameMapping(l string) string {
 func intPtr(i int64) *int64 {
 	return &i
 }
+
+func contains(arr []string, e string) bool {
```

Comment:
FYI, I tried to use `slices.Contains` from Go 1.18, but the golint failed in CI. Rolled back to this function for now. I guess we need to wait for this fix https://github.com/golangci/golangci-lint/issues/2649.